fil_finder/utilities.py

```diff
@@ -140,7 +140,8 @@ def distance(x, x1, y, y1):
 
 def padwithzeros(vector, pad_width, iaxis, kwargs):
     vector[:pad_width[0]] = 0
-    vector[-pad_width[1]:] = 0
+    if pad_width[1] > 0:
+        vector[-pad_width[1]:] = 0
```

Comment:
Would this just pad the first dimension when something like ```(2, 0)``` is given? Everywhere in the code should have```pad_width[0] == pad_width[1]```.